src/flamenco/repair/test_repair.c

```diff
@@ -0,0 +1,402 @@
+#include "../fd_flamenco_base.h"
+#include "fd_repair.h"
+#include "../../util/fd_util.h"
+#include <string.h>
+
+/* init repair test */
+static fd_repair_t *
+test_repair_setup( void ) {
+  ulong  footprint = fd_repair_footprint();
+  void * shmem     = aligned_alloc( fd_repair_align(), footprint );
+  FD_TEST( shmem );
+  fd_repair_t * repair = fd_repair_join( fd_repair_new( shmem, 14919811UL  ) );
+  FD_TEST( repair );
+  return repair;
+}
```

Comment:
it would have been better to track the credits with this API so you can test them together here